client/web/reporter/src/components/EditorToolbar.vue

```diff
@@ -28,28 +28,26 @@
         size="lg"
         size-confirm="lg"
         variant="danger"
-        :disabled="deleteDisabled || processingDelete || processing"
```

Comment:
You need to keep the processing otherwise the other buttons are clickable when saving is processing.